source/Activated.js

```diff
@@ -50,14 +57,17 @@ const Activated = ({
   // ========================================
   // = TELL PARENT THAT WE'RE ALL DONE HERE =
   // ========================================
+
   useEffect(() => {
     // @TODO: sync with transition duration?
     const unloadTimeout = isUnloading ? setTimeout(onDeactivate, 300) : null
 
+    document.addEventListener('keydown', handleEscClick, false)
```

Comment:
In the same area where we `clearTimeout(unloadTimeout)`, we also need to clean up this event listener:
```js
document.removeEventListener('keydown', handleEscClick, false)
```

Reply:
Applied!